connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -33,7 +35,7 @@ def jdk_create_jar(source_dir, files, jar_filename, dest_dir):
     if not check_jdk_environ_variable(JAR_EXECUTABLE_NAME):
         return False
 
-    abs_source_path = os.path.abspath(source_dir)
+    abs_source_path = source_dir.resolve()
```

Comment:
I like this, looks cleaner. Does it have the same functionality as "os.path.abspath(source_dir)", and is it the pythonic way forward?

Reply:
It's a `Path` method that does the same thing as `os.path.abspath`! 😁